gen/installer/aws.py

```diff
@@ -139,6 +140,10 @@ def get_test_session(config=None):
 
 
 def get_cloudformation_s3_url():
+    # TODO(lingmann): Remove this hack and figure out how to calculate properly...
```

Comment:
So ideally we should prbably pass in get_cloudformation_s3_url.

Options for the hacky method, easiest is probably to use monkeypatching to change the implementation of this function to just return the `config['cloudformation_s3_url']` that a user passes in.